dream_layer_backend/dream_layer.py

```
@@ -12,6 +12,10 @@
 from dream_layer_backend_utils.random_prompt_generator import fetch_positive_prompt, fetch_negative_prompt
 from dream_layer_backend_utils.fetch_advanced_models import get_lora_models, get_settings, is_valid_directory, get_upscaler_models, get_controlnet_models
 from dream_layer_backend_utils.random_prompt_generator import fetch_positive_prompt, fetch_negative_prompt
+import sys
```

Ruff (0.12.2)

Redefinition of unused `sys` from line 2

Remove definition: `sys`

(F811)